examples/policy/sandbox-automount-token-binding.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: admissionregistration.k8s.io/v1
```

Comment:
Would you please add a simple README.md ?

Reply:
Given that this is merged, I'll add one in a follow-up PR!